lang/gcc10-bootstrap/Portfile

```diff
@@ -176,20 +180,25 @@ if {${os.major} >= 18 && ${configure.sdkroot} ne ""} {
     configure.args-append --with-sysroot="[regsub {MacOSX1[0-9]\.[0-9]+\.sdk} ${configure.sdkroot} {MacOSX.sdk}]"
 }
 
-# clang (as) from Xcode 12.5 has various problems with gcc build
-if { ${os.platform} eq "darwin" && \
-         ( [ vercmp ${xcodeversion} 12.5 ] >= 0 || [ vercmp ${cltversion} 12.5 ] >= 0 ) } {
-    pre-configure {
-        ui_warn "Applying '--without-build-config' workaround to Xcode ${xcodeversion} / CLT ${cltversion}"
-        ui_warn "If versions > 12.5 please check if it is still required"
-    }
-    # gcc has build issues on macOS 11.3 with the use of Xcode clang as 'as'
-    # https://gcc.gnu.org/bugzilla/show_bug.cgi?id=100340
-    # https://trac.macports.org/ticket/62775
-    configure.args-append  --without-build-config
+if {${os.major} < 18 || ${os.major} > 19} {
+    default_variants    +universal
```

Comment:
Why? does enable `+universal` on `os.major` with value `18` brokes anything?

Reply:
after a lot of attempts to remember why I feel that the only reason is attemplt to make port file simpler.

Reply:
after a lot of attempts to remember why I feel that the only reason is attemplt to make port file simpler.